README.md

```diff
@@ -245,6 +245,16 @@ either you or they can follow these steps:
 
 This section covers the integration of GPT-4 Vision with Azure AI Search. Learn how to enhance your search capabilities with the power of image and text indexing, enabling advanced search functionalities over diverse document types. For a detailed guide on setup and usage, visit our [Enabling GPT-4 Turbo with Vision](docs/gpt4v.md) page.
 
+### Enabling Integrated Vectorization
+Azure AI search has integrated vectorization in preview https://techcommunity.microsoft.com/t5/ai-azure-ai-services-blog/announcing-the-public-preview-of-integrated-vectorization-in/ba-p/3960809#:~:text=Integrated%20vectorization%20is%20a%20new%20feature%20of%20Azure,pull-indexers%2C%20and%20vectorization%20of%20text%20queries%20through%20vectorizers.
+
+To enable integrated vectorization with this sample
+
+1. Delete existing index
+2. Run ```azd env set USE_FEATURE_INT_VECTORIZATION true```
+3. Run ```azd up``` to update system and user roles
+4. You could view the resources such as the indexer, skillet in azure portal and monitor the status
```

Comment:
Do they actually need to do a full up, or would `provision` work if they already deployed?

Reply:
They need a full up to assign new managed identites.